.babelrc

```diff
@@ -0,0 +1,20 @@
+{
+  "plugins": [
+    "lodash",
+    "transform-object-rest-spread",
+    "transform-es2015-modules-commonjs",
+    "transform-class-properties",
+    "array-includes"
+  ],
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "node": "8.0"
```

Comment:
Why not `"node": "current"`?

https://babeljs.io/docs/plugins/preset-env/

Reply:
@LordSputnik I used the setting for this same as in bookbrainz-data-js. I'll change it.